### ISSUE TITLE
New (or no) captcha

### ISSUE THREAD

**Issue by dostro** [dfinity/internet-identity#915]
The problem:
A substantial number of users have been complaining about the captcha working unreliably on first attempt. Also people hate captchas.

Discussion suggestions:
- Improve first-use reliability
- More user-friendly captcha than text input
- Implement [private access tokens](https://www.ietf.org/archive/id/draft-private-access-tokens-01.html) so that users on supported devices/browsers won't need a captcha at all

**Comment by nmattia** [comment]
> A substantial number of users have been complaining about the captcha working unreliably on first attempt. Also people hate captchas.

Yeah, this is quite bad. The CAPTCHA was introduced as a stop-gap measure until "People Parties", but ... yeah. I think it's time we look for a more long-term solution.

> Implement [private access tokens](https://www.ietf.org/archive/id/draft-private-access-tokens-01.html) so that users on supported devices/browsers won't need a captcha at all

Thanks for the great read! Didn't know of those. Support still seems to be limited but I'll try to dig out more info.
